napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -887,7 +887,7 @@ def test_background_color(qtbot, qt_viewer: QtViewer, dtype):
     data = np.zeros((10, 10), dtype=dtype)
     data[5:] = 10
     layer = qt_viewer.viewer.add_labels(data, opacity=1)
-    color = layer.colormap.map(10)[0] * 255
+    color = layer.colormap.map(10) * 255
```

Comment:
As you know, I like this change, but when I tried to do it you told me that it might be a breaking change for some users... I don't mind going in this direction either way — it is an easy fix for downstream users and matches the documentation.